pkg/graveler/committed/merge_iterator_test.go

```diff
@@ -246,18 +248,20 @@ func TestMergeSeek(t *testing.T) {
 	for _, tst := range tests {
 		t.Run(fmt.Sprintf("seek to %s", tst.seekTo), func(t *testing.T) {
 			it.SeekGE([]byte(tst.seekTo))
-			if it.Value() != nil {
-				t.Fatalf("value expected to be nil after SeekGE. got=%v", it.Value())
+			val, _ := it.Value()
```

Comment:
Shouldn't we also test the returned ranges?

Reply:
Added another test for merge seek with ranges